pkg/api/types/build_types.go

```diff
@@ -16,8 +16,13 @@
 
 package types
 
-// BuildCommandOptions specifies options for `nerdctl (image/builder) build`.
-type BuildCommandOptions struct {
+import "io"
+
+// BuildOptions specifies options for `nerdctl (image/builder) build`.
+type BuildOptions struct {
```

Comment:
```suggestion
type BuilderBuildOptions struct {
```

Probably the file name should be `builder_types.go`